src/style.css

```diff
@@ -1,3 +1,62 @@
+
 body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
   margin: 0;
 }
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.navbar {
+  margin: auto 0;
+  padding-right: 50px;
+}
+
+.navbar_list {
+  padding: 0;
+  display: flex;
+}
+
+.navbar_list li {
```

Comment:
do not use tags for styling elements. instead use classes
it can bu use maybe for html, body tags or reset some default styles